### PR TITLE
Fixes CPURuntime for MSVC.

### PR DESCRIPTION
**Pull request by lygstate**
*Description*:
Fixes CPURuntime for MSVC using custom cmake commands.

Related to https://github.com/pytorch/glow/issues/1144

### CODE REVIEW

File: lib/Backends/CPU/CMakeLists.txt [hunk 1]
```diff
@@ -9,43 +9,34 @@ find_program(LLVM_LINK_BIN
...
-                        PREFIX ""
-                        SUFFIX ""
-                        RULE_LAUNCH_COMPILE "${CLANG_BIN} <DEFINES> <INCLUDES> <FLAGS> -o <OBJECT> -c <SOURCE> #")
-target_compile_options(CPURuntime
-                       PRIVATE
-                         -ffast-math
-                         -g0
-                         -emit-llvm
-                         -O0)
+add_custom_target(CPURuntime
```
Comment (bertmaher) [inline lib/Backends/CPU/CMakeLists.txt]: Using `add_custom_target` means that CPURuntime will always get rebuilt, even if its deps have not changed.  Please try to find a way around that.

Comment (ayermolo) [reply lib/Backends/CPU/CMakeLists.txt]: I don't think it's quite correct. the CPURuntime will be evaluated, but it won't rebuild the .o files unless there is some change. Like they have been deleted., or the files that are compiles are modified. At least this is the behavior I am seeing on windows.

**Comment by bertmaher** [comment]
This PR is over a month old and possibly abandoned, so I'm going to close it out for now.  Feel free to reopen or open a new one with reviewers' comments addressed.  Thanks!

File: lib/Backends/CPU/CMakeLists.txt (hunk 1 above)
Comment (lygstate) [reply lib/Backends/CPU/CMakeLists.txt]: @bertmaher Do you have a look at ayemolo's comment? There is no other way to building CPURuntime for Windows.

**Pull request closed without merging**